index.html

```diff
@@ -131,6 +131,32 @@ <h2 class="connect">LET'S CONNECT</h2>
             </ul>
             </div>
         </section>
+        <Section class="form-Container">
```

Comment:
### OPTIONAL 

This is a suggestion rather than a requirement because the last reviewers didn't mention it 👀 

Be sure to find out why your design is looking a certain way by digging in your CSS  🧐 In this case, the following margins are not helping your design because your body has a fixed margin of 8px towards the full size of the screen 😵

![image](https://user-images.githubusercontent.com/15807343/154363165-6102d5ac-99e8-4948-8f0f-d8c22200e339.png)

Reply:
how do i fix this @mateo951 ?  not sure how to do it.

Reply:
On your CSS you could try something like...

    body {
        margin: 0
    }

This will replace the default margin for your body 🤓 Message me on Slack if you still encounter issues! Happy to help